bofire/data_models/domain/domain.py

```diff
@@ -150,11 +151,12 @@ def validate_constraints(self):
         """
 
         keys = self.inputs.get_keys()
-        for c in self.constraints:
-            if isinstance(c, LinearConstraint) or isinstance(c, NChooseKConstraint):
-                for f in c.features:
-                    if f not in keys:
-                        raise ValueError(f"feature {f} in constraint unknown ({keys})")
+        for c in self.constraints.get(
+            [LinearConstraint, NChooseKConstraint, MultiLinearConstraint]
+        ):
+            for f in c.features:  # type: ignore
```

Comment:
Why do you need to ignore this type while you didn't need to before?

Reply:
It is needed here, because there are constraint types which do not have a features attribute.